### PR TITLE
Add subscription protos

### PR DESCRIPTION
**Pull request by zalbiraw**
## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/getting-started/key-concepts/graphql-subscriptions.md [hunk 1]
```diff
@@ -17,11 +17,24 @@ In Tyk subscriptions are using the [WebSocket transport](https://developer.mozil
...
+
+Depending on whether you want to configure GraphQL proxy-only, UDG, or GraphQL Federation there are different places for the configuration option.
+
+The values for subscription types are the same on all API types:
+
+| Subscription Type                | Value inside API definition |
+|----------------------------------|-----------------------------|
+| WebSocket `graphql-ws`           | graphql-ws _default_        |
+| WebSocket `graphql-transport-ws` | graphql-transport-ws        |
+| Server-Sent Events (SSE)         | sse                         |
```
Comment (dcs3spp) [inline tyk-docs/content/getting-started/key-concepts/graphql-subscriptions.md:92]: Good overview added for the new section for how to setup new subscription types via API definition. 

Maybe in the table add another column that shows what configuration (GraphQL proxy-only, UDG, GraphQL Federation) each subscription type can map to?

Add example API definitions for graphql-ws, graphql-transport-ws and SSE

**Review by dcs3spp — CHANGES_REQUESTED** [review]

File: tyk-docs/content/getting-started/key-concepts/graphql-subscriptions.md (hunk 1 above)
Comment (zalbiraw) [reply tyk-docs/content/getting-started/key-concepts/graphql-subscriptions.md:92]: > Maybe in the table add another column that shows what configuration (GraphQL proxy-only, UDG, GraphQL Federation) each subscription type can map to?

Yes, that is a great idea. I will add that.

> Add example API definitions for graphql-ws, graphql-transport-ws and SSE

We can add an example of the section but these are just the possible values to a single field. Add a whole API definition will be an overkill and maybe confusing

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**